src/docs/common/callouts/callout-each-property.adoc

```diff
@@ -0,0 +1 @@
+The `@EachProperty` annotation creates a `ConfigurationProperties` bean for each sub-property within the given property.
```

Comment:
```suggestion
The `@EachProperty` annotation creates a `ConfigurationProperties` bean for each sub-property within the given name.
```

Reply:
@alvarosanchez I think we should change it in core as well https://docs.micronaut.io/latest/guide/#eachProperty. Thoughts?

Reply:
Fine to me. It's just to not have the word "property" 3 times.